metadata-db-app/src/cmr/metadata_db/data/oracle/search.clj

```diff
@@ -82,6 +83,30 @@
     ;; add index hint to the generated sql statement
     (update-in stmt [0] #(string/replace % #"SELECT" (format "SELECT /*+ INDEX(%s) */" table)))))
 
+(defmethod gen-find-concepts-in-table-sql :subscription
+  [concept-type table fields params]
+  (let [fields (cons :last_notified_at fields)
+        root-query (if (:include-all params)
```

Comment:
Nitpick: I usually try to break things up a bit when I have an `if` statement in a `let` block. If it makes sense to, maybe put this `if` into its own function